testing/internal/e2e/boundary/scope.go

```diff
@@ -45,7 +47,7 @@ func CreateNewOrgCli(t testing.TB, ctx context.Context) string {
 	output := e2e.RunCommand(ctx, "boundary",
 		e2e.WithArgs(
 			"scopes", "create",
-			"-name", "e2e Org",
+			"-name", fmt.Sprintf("e2e Org %s", time.Now().String()),
```

Comment:
Was this added to avoid conflicts? When do we create 2 orgs at the same time? Adding this would make it harder to detect when a test doesn't clean up properly (since it won't conflict anymore).